extensions/wikia/Recirculation/RecirculationHooks.class.php

```diff
@@ -100,51 +104,57 @@ static public function canShowDiscussions( $cityId, $ignoreWgEnableRecirculation
 	}
 
 	private static function addLiftIgniterMetadata( OutputPage $outputPage ) {
-		global $wgCityId,
-		       $wgDevelEnvironment,
-		       $wgEnableArticleFeaturedVideo,
-		       $wgIsPrivateWiki,
-		       $wgLanguageCode,
-		       $wgStagingEnvironment,
-		       $wgWikiaEnvironment;
+		$metaData = self::getMetaData();
+		$metaDataJson = json_encode( $metaData );
 
+		$outputPage->addScript(
+			"<script id=\"liftigniter-metadata\" type=\"application/json\">${metaDataJson}</script>"
+		);
+	}
+
+	private static function getMetaData() {
+		global $wgLanguageCode, $wgCityId, $wgIsPrivateWiki, $wgEnableArticleFeaturedVideo;
 		$context = RequestContext::getMain();
 		$title = $context->getTitle();
 		$articleId = $title->getArticleID();
-		$metaData = [];
 		$metaDataService = new LiftigniterMetadataService();
 		$metaDataFromService = $metaDataService->getLiMetadataForArticle( $wgCityId, $articleId );
-
+		$title = $context->getTitle();
 		$metaData['language'] = $wgLanguageCode;
+		$isProduction = self::checkIfIsProduction();
+		$isPrivateWiki = WikiFactory::isWikiPrivate( $wgCityId ) || $wgIsPrivateWiki;
+		$shouldNoIndex = !$isProduction ||
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce the number of conditional operators (4) used in the expression (maximum allowed 3). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1067)